test/unit.js

```diff
@@ -1,21 +1 @@
-describe('XMLHttpRequest Compatibility: event [load]', function() {
-    //var xhr = new ChromeSocketsXMLHttpRequest();
-    var XhrClass = require('freedom-xhr').coretcpsocket;
-    var xhr = new XhrClass();
-    console.log(xhr);
-
-    beforeEach(function(done) {
-        xhr.addEventListener('load', function (e) {
-            done();
-        });
-    });
-
-    xhr.open('GET', 'http://httpconsole.com/text/Hello%20World');
-    xhr.send(null);
-
-    //@todo - this seems like more of an integration test?
-    it ('should work get a successful response', function (done) {
-        expect(xhr.readyState).toBe(xhr.DONE);
-        done();
-    });
-});
+// TODO: Add unit tests
```

Comment:
This test never passed, maybe because it doesn't properly load freedom.  It's also an integration test, not a unit test, and is redundant with the integration tests in our integration test framework (which do pass).  To make a unit test, I think we need to add a mock freedom socket API.